wcfsetup/install/files/lib/event/moderation/DeletedItemsCollecting.class.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace wcf\event\moderation;
+
+use wcf\data\object\type\ObjectTypeCache;
+use wcf\event\IPsr14Event;
+use wcf\page\DeletedContentListPage;
+use wcf\system\moderation\DeletedItems;
+use wcf\system\request\LinkHandler;
+
+/**
+ * Requests the collection of deleted item types.
+ *
+ * @author      Marcel Werk
+ * @copyright   2001-2025 WoltLab GmbH
+ * @license     GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @since       6.2
+ */
+final class DeletedItemsCollecting implements IPsr14Event
+{
+    /**
+     * @var DeletedItems[]
```

Comment:
```suggestion
     * @var list<DeletedItems>
```

The difference is that `list<…>` guarantees that this is a continuous zero-based array.